chapters/7-relationships-between-SPDX-elements.md

```diff
@@ -2,16 +2,23 @@
 
 ## 7.1 Relationship <a name="7.1"></a>
 
-**7.1.1** Purpose: This field provides information about the relationship between two SPDX elements. For example, you can represent a relationship between two different Files, between a Package and a File, between two Packages, or between one SPDXDocument and another SPDXDocument. The relationships between two elements that are supported are:
+**7.1.1** Purpose: This field provides information about the relationship between two SPDX elements. For example, you can represent a relationship between two different Files, between a Package and a File, between two Packages, or between one SPDXDocument and another SPDXDocument. 
+
+In cases where there are "known unknowns", the use of the keyword `NOASSERTION` can be used on the right hand side of a relationship to indicate that the author is not asserting whether there are other SPDX elements (package/file/snippet) are connected by relationships or not.  ie. There could be some, but the author is not asserting one way or another.  
+
+Similarly, the use of the keywords `NONE` can be used to indicate that an SPDX element (package/file/snippet) has no other elements connected by some relationship to it.
+
+The relationships between two SPDX elements that are supported are:
 
 | Relationship           | Description | Example |
 |------------------------|-------------|---------|
 | DESCRIBES              | Is to be used when SPDXRef-DOCUMENT describes SPDXRef-A                                               | An SPDX document `WildFly.spdx` describes package ‘WildFly’. Note this is a logical relationship to help organize related items within an SPDX document that is mandatory if more than one package or set of files (not in a package) is present. |
 | DESCRIBED_BY           | Is to be used when SPDXRef-A is described by SPDXREF-Document                                         | The package ‘WildFly’ is described by SPDX document `Wildfly.spdx`. |
 | CONTAINS               | Is to be used when SPDXRef-A contains SPDXRef-B.                                                      | An ARCHIVE file `bar.tgz` contains a SOURCE file `foo.c`. |
 | CONTAINED_BY           | Is to be used when SPDXRef-A is contained by SPDXRef-B.                                               | A SOURCE file `foo.c` is contained by ARCHIVE file `bar.tgz` |
-| DEPENDENCY_MANIFEST_OF | Is to be used when SPDXRef-A is a manifest file that lists a set of dependencies for SPDXRef-B        | A file `package.json` is the dependency manifest of a package `foo`. Note that only one manifest should be used to define the same dependency graph. | 
+| DEPENDS_ON             | Is to be used when SPDXRef-A depends on SPDXRef-B. | Package A depends on the presence of package B in order to build and run |
```

Comment:
I see why it might sound better to add this. However, my worry is that because we've added a bunch of different *_DEPENDENCY_OF types (e.g., BUILD, DEV, etc), adding a new reciprocal one here would suggest that we'd also need reciprocal versions of each of those.

Can we reach the same outcome by keeping just the existing one-way relationship types, but flipping around the ordering of elements? e.g.:

`NONE DEPENDENCY_OF SPDXRef-B`

`NOASSERTION BUILD_DEPENDENCY_OF SPDXRef-B`

It might not flow quite as well as having the reciprocal versions, but I worry that the additon makes it more complex for tools to have to handle both cases of every reciprocal relationship type when we add new reciprocal ones.

Reply:
See @goneall's comment in the issue about problems if they aren't on the RHS of the relationship.     
Rather than going into all the variants of dependencies, mirroring the most generic version seemed the best compromise.